scripts/test/MultiPlotting/MultiPlottingContext_test.py

```diff
@@ -70,5 +71,31 @@ def test_updateLayout(self):
             patch.assert_called_with(gridspec,figure,2)
 
 
+    def test_subplotEmptyTrue(self):
+        names = ["one","two","three"]
+        for name in names:
+            self.context.addSubplot(name, mock.Mock())
+
+        for name in names:
+             self.assertEquals(self.context.is_subplot_empty(name),True)
```

Comment:
As a very minor point as of python 3.3 assertEquals is deprecated in favor of assertEqual for now both work equally well but might be worth using the singular for new code so as to me more forward compatible.